CHANGELOG.rst

```diff
@@ -17,6 +17,12 @@ Unreleased
 ----------
 None
 
+[3.49.0]
+--------
+fix: Return None for context if a role assignment has no enterprise_customer and does not apply to all contexts.
```

Comment:
so actually, hmm, ENT-4348 adds validation to prevent us from being able to save role assignments that have no enterprise_customer and that do not apply to all contexts 🤔 . also as part of ENT-4349, we've removed all existing system wide role assignments that meet this criteria too

Reply:
so no need for this anymore?

Reply:
sure. and yeah, leave this in the changelog cause as of this commit it'll still be possible